feature LP132-missing-modifier-as-parameter

Lupillo17 opened the pull request:
Add modifier as parameter to the component function.

Lupillo17 committed:
Add modifier as parameter to the component function.

carlosph816 reviewed (approved):
Well done!

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:
```diff
@@ -78,28 +78,30 @@ fun LpPostCard(
+) {
     Card(
         modifier = modifier
             .fillMaxWidth(sizeWidth)
```
This modifier property `fillMaxWidth(sizeWidth)` shouldn't be on this modifier, the function that implement this component is the only one that defines this kind of sizes. So you could remove this property

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:
```diff
@@ -78,28 +78,30 @@ fun LpPostCard(
-        ){
+        ) {
             Row {
                 Column {
```
You can remove this column

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:
```diff
@@ -78,28 +78,30 @@ fun LpPostCard(
-                    Box(modifier = Modifier
-                        .fillMaxWidth(0.20f)
-                        .padding(0.dp)) {
+                    Box(
```
you can remove this box and just have the image and the column in this row.
In the column you can add in modifier the property padding start to have separation between the image and all the texts.

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:
```diff
@@ -126,14 +128,14 @@ fun LpPostCard(
                 }
             }
-            Row(modifier = Modifier.padding(10.dp)){
+            Row(modifier = Modifier.padding(10.dp)) {
```
You can remove this `Row` and move this modifier to the text

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:
```diff
@@ -126,14 +128,14 @@ fun LpPostCard(
+            Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.End) {
                 IconButton(onClick = onCommentClick) {
                     Icon(
                         Icons.Outlined.Comment,
```
Inside of the Icon, you can remove the parameter modifier as is not being used. This also apply for the modifier of the Icon favorite

melissaquezada94 commented on core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt:
```diff
@@ -167,15 +170,13 @@ fun LpPostCard(
-        modifier = Modifier
-            .height(120.dp)
-            .heightIn(120.dp, 250.dp)
-            .padding(8.dp),
```
you can add padding property to the modifier `.padding(8.dp)`, you

Lupillo17 committed:
Merge branch 'develop' into feature/LP132-missing-modifier-as-parameter

Lupillo17 committed:
Changes to LpPostCard
- remove parameter sizeWidth
- remove property fillMaxWidth(sizeWidth) from Card
- remove unnecessary Row, Box and Column in all the component
- remove unnecessary paddings
- fixing some modifier paddings
- remove unnecessary modifiers

melissaquezada94 commented on app/src/main/java/com/iteneum/leasepert/MainActivity.kt:
```diff
@@ -4,13 +4,14 @@ import android.os.Bundle
         setContent {
             LeasePertTheme {
-
+                ExampleView()
```
If you are not going to use this function please delete it and the import as well

Lupillo17 committed:
Merge branch 'develop' into feature/LP132-missing-modifier-as-parameter

Lupillo17 committed:
Remove function and imports from MainActivity.kt for "ExampleView"
